packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -57,6 +59,8 @@ - (instancetype)initWithURL:(NSURL *)url
 static void *playbackBufferEmptyContext = &playbackBufferEmptyContext;
 static void *playbackBufferFullContext = &playbackBufferFullContext;
 
+AVPictureInPictureController *pipController;
```

Comment:
I think we discussed about this last time (sorry if i missed your respond). I think this can be marked as static to scope it under the current translation unit (just like the previous line). Or you can make it a property of the plugin.

Reply:
I fixed this one as well